README.md

```diff
@@ -17,4 +17,4 @@ This task is similar to [Static List of TODOs](https://github.com/mate-academy/r
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_static-list-of-posts/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://RomeoKs.github.io/react_static-list-of-posts/) and add it to the PR description.
```

Comment:
@RomeoKs i can't see the deployed page 
<img width="1350" alt="no gh pages deployments" src="https://user-images.githubusercontent.com/39308717/210315383-2420cf7f-6eb6-4d06-a40c-5eace3c5a67e.png">